modules/Bio/EnsEMBL/Compara/PipeConfig/PrepareMasterDatabaseForRelease_conf.pm

```diff
@@ -71,6 +71,7 @@ sub default_options {
         'prev_dbs'            => ['*_prev'],
         'taxonomy_db'         => 'ncbi_taxonomy',
         'incl_components'     => 1, # let's default this to 1 - will have no real effect if there are no component genomes (e.g. in vertebrates)
+        'patch_db_exe'        => $self->check_exe_in_ensembl('ensembl-compara/scripts/production/patch_database.pl'),
```

Comment:
This addition is also required in `BuildNewMasterDatabase_conf.pm`, so I would suggest moving this line to `ENV.pm::executable_locations()`, so you don't need to edit either pipeline.

Reply:
Thank you. Done. Will commit together with other changes.